src/Geosuggest.jsx

```diff
@@ -11,6 +11,8 @@ import filterInputAttributes from './filter-input-attributes';
 import Input from './input';
 import SuggestList from './suggest-list';
 
+import * as Nominatim from 'nominatim-browser';
```

Comment:
Requiring this even though another geocoding service is used bloats the component.